unit_test/blas/Test_Blas2_gemv.hpp

```diff
@@ -51,17 +51,17 @@ void impl_test_gemv(const char* mode, int M, int N) {
 
   {
     ScalarX randStart, randEnd;
-    Test::getRandomBounds(10.0, randStart, randEnd);
+    Test::getRandomBounds(1.0, randStart, randEnd);
```

Comment:
@vqd8a: Just change the bound for float?

Reply:
What do you think?

Reply:
Given that this is used for filling the test matrices and vectors I think 1.0 is fine for checking correctness.

Reply:
Yes, this is fine with me.

Reply:
@vqd8a In my own build, it looked like changing the range to [-1.0, 1.0] wasn't enough to make it pass. I think what's happening is that some rows add up to a number close to 0, so the relative error is more sensitive. Changing the random range to [0, 1] worked.

Reply:
@brian-kelley Yes, I see the same thing. The random range [0, 1] works or we can reduce more the eps from 1e-3 to 1e-2
```double eps = (std::is_same<typename KAT_Y::mag_type, float>::value ? **1e-2** : 5e-10);```
This works for [-1.0, 1.0]
What do you guys think?

Reply:
Actually, changing to 1e-2 works for the previous range [-10,10]